exporter/otlphttpexporter/factory_test.go

```diff
@@ -62,9 +63,10 @@ func TestCreateTracesExporter(t *testing.T) {
 	endpoint := "http://" + testutil.GetAvailableLocalAddress(t)
 
 	tests := []struct {
-		name     string
-		config   Config
-		mustFail bool
+		name            string
+		config          Config
+		mustFail        bool
```

Comment:
s/mustFail/mustFailOnCreate

Reply:
Thanks! fixing it. Changing in other PRs as well.